server.py

```diff
@@ -36,17 +36,25 @@ def index():
 ######################################################################
 @app.route('/products', methods=['GET'])
 def list_products():
```

Comment:
Did you run the unit tests to ensure that the changes do not break the current implementation?

Also, please add test cases to validate the functionality of the `search` URL attribute. If you need help writing test cases we can schedule a hangout session or meet in person to do so.